internal/config/client.go

```diff
@@ -31,7 +31,7 @@ const (
 type MongoDBClient struct {
 	Atlas           *matlasClient.Client
 	AtlasV2         *admin.APIClient
-	AtlasV220231115 *admin20231115.APIClient
+	AtlasV220240530 *admin20240530.APIClient // used in advanced_cluster and cloud_backup_schedule for avoiding breaking changes
```

Comment:
just double checking. This is correct right? @AgustinBettati

Reply:
Yes, with 2024-08-05 released (and used for AtlasV2 client), 2024-05-30 is the nearest SDK version that still consumes the older versions.